### DISCUSSION TITLE
How to craeate USB if NO MAC

### DISCUSSION THREAD

**Discussion by afzl-wtu** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#46] [Ideas]
First of all I am very grateful to you, I just find this guide today on internet before that I installed Catalina by following opencore official guide 5 days ago SMBIOS MacBookPro10,1.  This guide is not ranked on google sad, otherwise I follow it first and did not waste my time on opencore official guide please google searches and trying hard to make battery patches (still no battery patch) but Thank God I have just found this guide today.

As I said earlier I have hackintosh with Catalina with SMBIOS MAcBookPro10,1.. So when I open App Store and try to install "macos Big Sur.app"  it opens My Catalina setting app and say no update available. The only thing I see in my mac update section is Somekind of Supplementry Update to 15.7.1 of 1 GB. 

So now I am trying to change my SMBIOS to MacBookPro11,1 according to one of your comment hope it don't break my Catalina. There should be some way to install only recovery in USB like opencore windows guide and then it should be processed under you installer.

**Comment by chris1111** [comment]
Thanks 
Your subject name is create USB if no Mac I read you and you have a Mac?

If you need Big Sur download from Apple server just use this [Download Big Sur](https://github.com/chris1111/Download-BigSur) Its about 25 minutes with normal internet connection so be patient

**Reply by afzl-wtu** [reply]
Ok everything fine now, I just asked if a person does not have any Mac or Hackintosh. I want a Windows installer like opecore official guide.
